src/components/Menu-Badge/MenuBadge.tsx

```diff
@@ -0,0 +1,100 @@
+import { Box, Typography } from '@mui/material';
+
+import { COLOR } from './MenuBadge.constraints';
+
+type MenuBadgeProps = {
+  label: string;
+  variant?: 'default' | 'color';
+};
+
+export const MenuBadge: React.FC<MenuBadgeProps> = ({ label, variant = 'default' }) => {
+  const isColor = variant === 'color';
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        gap: '1.8rem',
+        background: 'transparent',
+        cursor: 'pointer',
+        padding: '6px 12px',
+      }}
+      data-testid="menuBadge"
+    >
+      <Box
```

Comment:
Using a <Box component='svg'> for the icon is OK, but the SVG defines a `<linearGradient id="gradient">` with a static id. If multiple MenuBadge instances are mounted, the id will collide leading to incorrect gradients.

**Sugerencia:**
```typescript
Generate a unique id per instance using `useId()` (React 18) or a small uniqueId util and use it for the gradient id. Example:

const id = useId();
<linearGradient id={`menuBadgeGradient-${id}`}> ... </linearGradient>
stroke={isColor ? `url(#menuBadgeGradient-${id})` : theme.palette.text.primary}

```